src/core/server/http/base_path_proxy_server.ts

```diff
@@ -195,16 +197,17 @@ export class BasePathProxyServer {
           agent: this.httpsAgent,
           passThrough: true,
           xforward: true,
-          mapUri: (request: Request) => ({
```

Comment:
Could just make this function `async` too, which I personally prefer to using `Promise.resolve()`.